hardware_interface/include/hardware_interface/resource_manager.hpp

```diff
@@ -77,6 +80,10 @@ class HARDWARE_INTERFACE_PUBLIC ResourceManager
 
   ~ResourceManager();
 
+  void create_interface_value_publisher();
```

Comment:
Can you add some documentation to these methods?

Reply:
done